chirp/drivers/kenwood_live.py

```diff
@@ -121,13 +121,18 @@ def get_id(ser):
             ser.baudrate = i
             ser.write(LAST_DELIMITER[0].encode())
             ser.read(25)
-            resp = command(ser, "ID")
+            try:
+                resp = command(ser, "ID")
+            except UnicodeDecodeError:
+                LOG.info("Unknown utf-8 byte received at baud %i with \
+                        delimiter \"%s\"" % (i, repr(delimiter)))
```

Comment:
This will just create noise in the logs for the common situation of talking to the radio at the wrong baud rate. I think we should just move on here. We return on success if L133 succeeds, but if not, we don't log "nope, wrong baud rate."